sessions/debugging.qmd

```diff
@@ -62,6 +230,29 @@ TODO: Reading text
 #| code-summary: "**Click for the solution**. Only click if you are struggling or are out of time."
 ```
 
+## Alternatives to the debugger
+
+Other debugging methods include adding 'breakpoints' or `print()`
```

Comment:
Using breakpoints is a big part of debugging in formal IDE environments, so I wouldn't suggest putting this within the "alternatives" section.